build.sbt

```diff
@@ -543,8 +548,9 @@ lazy val warnUnusedImport = Seq(
     CrossVersion.partialVersion(scalaVersion.value) match {
       case Some((2, 10)) =>
         Seq()
-      case Some((2, n)) if n >= 11 =>
+      case Some((2, n)) if ((n >= 11) && (n <= 12)) =>
```

Comment:
just a silly style thing, but you can `case Some((2, 11 | 12))`. I sometimes forget that `|` is usable other than at the top level of the match.

Reply:
I had no idea!

Reply:
There's also

```scala
import sbt.librarymanagement.{ SemanticSelector, VersionNumber }
VersionNumber(scalaVersion.value).matchesSemVer(SemanticSelector("2.11.x || 2.12.x"))
```